packages/@lwc/synthetic-shadow/src/polyfills/document-body-shadow/polyfill.ts

```diff
@@ -15,23 +15,12 @@ import {
     ArrayFind,
```

Comment:
HTMLBodyElement.prototype is being patched twice, once above in faus-shadow/element.ts(https://github.com/salesforce/lwc/pull/1322/files#diff-8bc8822d07cd6e658acf12ae727b6a76R422)

Reply:
fixed